### ISSUE TITLE
Data discrepancy for German category 1.A.2.g

### ISSUE THREAD

**Issue by simschul** [openclimatedata/unfccc-detailed-data-by-party#13]
Thanks for providing this nice and very helpful data package!

I have discovered one problem concerning the CO2 emission from the German category 1.A.2.g: Its only sub-category with reported emissions (1.A.2.g.vii) does not match the emissions reported under 1.A.2.g. The discrepancy is quite substantial e.g. in 2019 1.A.2.g reports 74947.20 kt while 1.A.2.g.vii only 3450.54 kt.

The numbers from the database you provide match those from the UNFCCC data interface, thus I assume a reporting error. Or do I miss something?

**Comment by rgieseke** [comment]
Good catch, no idea why that is. Maybe the remainder is assumed to be in 1.A.2.g.viii Other?

Have you compared with the CRF submission? E.g. via PRIMAP-crf 
https://doi.org/10.5281/zenodo.4723476

**Comment by simschul** [comment]
Just checked PRIMAP-crf. There 1.A.2.g seems to correspond to 1.A.2.m, but without a finer subdivision into sub-categories.

**Comment by rgieseke** [comment]
Yeah, you might let them know at

https://unfccc.int/process/transparency-and-reporting/greenhouse-gas-data/contact-0

**Comment by simschul** [comment]
I contacted UBA, and they confirmed that the remainder should be in 1.A.2.g.viii Other. 

The reason why it is not included in the submitted data is a software bug. A as far as I understood them correctly, every country has its own way of getting the data from their own database into the "CRF-Reporter" (= the official web-based system to report emissions in the CRF). The problem seems to be that the CRF-Reporter is not implemented the way the CRF tables were actually tuned. Unlike other countries, Germany did not find a workaround for this because of its special database system. (hope I cite them correctly :) )

**Comment by rgieseke** [comment]
Wow, interesting :-)

Thanks a lot for following up!
